template/data/fr/pages.json

```
@@ -0,0 +1,308 @@
+{
+  "pagesFR": [
+    {
+      "id": 5,
+      "slug": "fr",
+      "locale": "fr",
+      "localizations": [
+        {
+          "id": 1,
+          "locale": "en"
+        }
+      ],
+      "shortName": "Home",
+      "status": "published",
+      "metadata": {
+        "metaTitle": "Strapi corporate site starter",
+        "metaDescription": "Construire un site modifiable avec Strapi",
+        "twitterCardType": "summary",
+        "twitterUsername": null,
+        "shareImage": null
+      },
+      "contentSections": [
+        {
+          "__component": "sections.hero",
+          "title": "Le meilleure façon de faire votre site Corporate",
+          "label": "Nouveau Strapi starter",
+          "description": "Commencer vite avec votre site corporate",
+          "smallTextWithLink": "Vous voulez construire votre propre starter? Guide à venir",
+          "buttons": [
+            {
+              "url": "https://github.com/strapi/strapi-template-corporate",
+              "newTab": false,
+              "text": "Commencer",
+              "type": "primary"
+            },
+            {
+              "url": "https://github.com/strapi/strapi-template-corporate",
+              "newTab": true,
+              "text": "Voir le code",
+              "type": "secondary"
+            }
+          ],
+          "picture": null
+        },
+        {
+          "__component": "sections.feature-rows-group",
+          "features": [
+            {
+              "title": "Les équipes marketing peuvent maintenant faire leurs propre pages",
+              "description": "Grace à une liste pré-construite de blocs UI, les marketers peuvent faire la mise en page exactement comme ils/elles veulent",
```

Here's one way we can rephrase this to avoid the marketer anglicism, and avoid specifying the gender

```suggestion
              "description": "Grace à une liste pré-construite de blocs UI, les équipes marketing peuvent faire la mise en page exactement comme elles veulent",
```

Marketing has become widely used, but marketer still sounds awkward in french